custom-vu/src/main/kotlin/jces1209/vu/page/FalliblePage.kt

```diff
@@ -56,6 +56,14 @@ class FalliblePage private constructor(
             )
         )
 
+        fun serverErrors() = potentialErrors(
```

Comment:
I like how nicely it fits with the existing structure and semantics

Reply:
Yeah 😎